ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx

```diff
@@ -26,6 +27,15 @@ function displayCveType(cveType: string): string {
     }
 }
 
+export const sortFields = {
+    CVE: 'CVE',
+    CVEStatus: 'Cluster CVE Fixable',
+    CVEType: 'CVE Type',
+    CVSSScore: 'CVSS',
+};
```

Comment:
CVE Status field name taken from https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L95
CVE Type field name taken from https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L54

All of these were tested against staging, where there is more data, but unfortunately still only a single `Type` of CVE across cluster so I was unable to validate the data when sorting by `CVEType`.

Reply:
A thought, not necessarily a change request. Based on discussion at office hours, what is pro and con:

```ts
export const CVE_SORT_FIELD = 'CVE';
export const CVE_STATUS_SORT_FIELD = 'Cluster CVE Fixable';
export const CVE_TYPE_SORT_FIELD = 'CVE Type';
export const CVSS_SCORE_SORT_FIELD = 'CVSS';
```

Reply:
➕ I'm working on a Container-centralized sort options export file, complete with SCREAMING SNAKES as discussed at our meeting on Monday.